### DISCUSSION TITLE
egzumer "RX Only" edition

### DISCUSSION THREAD

**Discussion by bmn001** [egzumer/uv-k5-firmware-custom#563] [Ideas]
For those of us who use our devices as inexepensive scanners and have _no intention_ of ever transmitting **anything**, it might be cool to have a "RX only" version of egzumer.

I'm envisioning egzumer **with all the parts related to TX removed**.  This would _free up memory_ and potentially allow for features like a waterfall display, frequency lockouts, more than 2 scanning lists, and maybe even more than 200 memory slots.

Whaddya say?

**Comment by AubsUK** [comment]
If you look at the `Interesting Forks` at the top of the `Discussion` section, I've made a fork with 10 scan lists, permanent / temporary channel lockouts. No changes to the TX though, but that might be an option for future.

**Reply by bmn001** [reply]
Actually I was looking at yours a few days ago and was considering a switch to it!  Once you nail [this Chirp bug](https://github.com/AubsUK/uv-k5-firmware-custom/issues/28), I'm all over it. :)
